android/dosidicus_mobile/engine/simulation.py

```
@@ -294,12 +314,18 @@ def add_visitor(self, snapshot, name="a friend"):
             "vx": 60.0 if from_left else -60.0,
             "vy": random.uniform(-20, 20),
             "dir": "right" if from_left else "left",
-            "leave_at": now + self.VISIT_SECONDS,
+            "leave_at": now + (seconds if seconds is not None else self.VISIT_SECONDS),
             "leaving": False,
+            "stole": 0,
+            "ink_cd": 0.0,
+            "wants_rock": random.random() < 0.6,   # half the guests are here to loot a rock
```

Ruff (0.16.0)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)